src/reducers/__tests__/bitcoinReducer.test.js

```diff
@@ -152,4 +153,23 @@ describe('Bitcoin reducer', () => {
       });
     });
   });
+
+  describe(UPDATE_BITCOIN_TRANSACTIONS, () => {
+    const address = '<address>';
+    describe('for an existing address', () => {
+      it('adds the transactions', () => {
+        const updateTransactions: BitcoinReducerAction = {
+          type: UPDATE_BITCOIN_TRANSACTIONS,
+          transactions: [],
+          address,
+        };
+
+        const state = reducer(initialState, updateTransactions);
+
+        expect(reducer(state, updateTransactions)).toMatchObject({
+          data: { transactions: [] },
+        });
```

Comment:
What are we testing here? It looks like we should add some transactions to make sure this does what's expected to do

Reply:
Just verifying that the reducer exists. Please feel free to fill it up with mock data